Source/WebCore/Modules/fetch/FetchHeaders.cpp

```diff
@@ -68,7 +68,11 @@ static ExceptionOr<void> appendToHeaderMap(const String& name, const String& val
         return canWriteResult.releaseException();
     if (!canWriteResult.releaseReturnValue())
         return { };
-    headers.set(name, combinedValue);
+
+    if (equalIgnoringASCIICase(name, "Set-Cookie"_str))
```

Comment:
We might want to special case set-cookie a bit earlier.
The computation of the combined value is done using headers and not setCookieValues for Set-Cookie.
I do not think we need to combine values for SetCookie validation.